src/index.html

```diff
@@ -9,6 +9,32 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header>
+      <nav class="nav">
+        <div class="left">
+          <img 
+          class="nav-img" 
+          src="images/logo.png" 
+          alt="squere"
+          >
+        </div>
+        <span class="links">
+          <a class="nav-link is-active" href="apple">Apple</a>
+          <a class="nav-link" href="samsung">Samsung</a>
+          <a class="nav-link" href="smartphones">Смартфоны и телефоны</a>
+          <a 
+            class="nav-link" 
+            href="notebooks"
+            data-qa="hover"
+          >
+            Ноутбуки и компьютеры
+          </a>
+          <a class="nav-link" href="gadgets">Гаджеты</a>
+          <a class="nav-link" href="palms">Планшеты</a>
+          <a class="nav-link" href="foto">Фото</a>
+          <a class="nav-link" href="video">Видео</a>
+        </span>
```

Comment:
use `<ul>` for group all your links and then `<li>` for ul items
make your markup semantic